Models/forum_post_model.py

```diff
@@ -0,0 +1,32 @@
+class ForumPostModel:
+    """
+    A model that contains info about a forum post.
+
+    ...
+
+    Attributes
+    ----------
+    thread_id : int
+        unique id for the thread
+    page_num : int
+        page number in the thread of the post
+    post_num : int
+        post number on the page
+    username : str
+        the username of the poster
+    post_body : str
+        the body message of the post
+    timestamp : str
+        the cleaned timestamp
+    scraped_timestamp : str
+        the raw timestamp as it was posted
+    """
+
+    def __init__(self, thread_id, page_num, post_num, username, post_body, timestamp, scraped_timestamp)
```

Comment:
this constructor is fine, but let's add a class method that can take an array. it may help with code readability
'@classmethod
def from_array(cls, arr):`